src/release_notes_workflow/component_release_notes.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+from typing import Any
+from manifests.input_manifest import InputComponentFromSource
+
+
+class ComponentReleaseNotes:
+
+    def __init__(self, component: InputComponentFromSource, build_version: str, root: str) -> None:
+        self.component = component
+        self.build_version = build_version
+        self.root = root
+        self.filename
+
+    @property
+    def filename(self) -> str:
+        return self._filename
+
+    @filename.setter
+    def filename(self, filename): 
+        self._filename = filename
+
+    @property
+    def path(self) -> str:
+        return os.path.join(self.root, "release-notes")
+
+    def path_exists(self) -> bool:
+        return os.path.exists(self.path)
+
+    def exists(self) -> bool:
+        print(self.filename)
+        return self.path_exists() and any(fname.endswith(self.filename) for fname in os.listdir(self.path))
+
+
+    def check(self) -> str:
+        if self.exists():
```

Comment:
Were you trying to return a third value when path does not exist? Otherwise you don’t need any of this code, just call `exists()` and include true/false in the table.

Reply:
Modified to just run `exists()`, output is expected to be `True/False`.